Source/OCMock/OCMNonRetainingObjectReturnValueProvider.m

```diff
@@ -38,6 +38,10 @@ - (void)handleInvocation:(NSInvocation *)anInvocation
     {
         // methods that "create" an object return it with an extra retain count
         [returnValue retain];
+    } else if([sel hasPrefix:@"init"]) {
```

Comment:
I realize the above checks have the same sorta issues, but this one especially makes me worry about properties/etc. that start with things like "initiary".  Maybe all these really should be checking that the are exactly that string or that the following character is uppercase to they method meet the ARC rules.